solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -730,6 +731,8 @@ public void load() {
 
     logging = LogWatcher.newRegisteredLogWatcher(cfg.getLogWatcherConfig(), loader);
 
+    StartupLoggingUtils.checkRequestLogging();
```

Comment:
I could have logged this already in `CoreContainerProvider` but I wanted to have the warn included in the Admin UI, and thus had to do it after initializing the `LogWatcher`. I wonder if we can initialize LogWatcher earlier on?

Reply:
Spun off #543 for earlier init